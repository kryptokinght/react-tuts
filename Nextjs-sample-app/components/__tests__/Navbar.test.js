import React from 'react';
import Navbar from '../Navbar';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json'; //removes unnecessary JSON from snapshot

describe('<Navbar />', () => {
    it('should be defined', () => {
        expect(Navbar).toBeDefined();
    });
    it('should render', () => {
        const wrapper = shallow(<Navbar />);
        expect(shallowToJson(wrapper))
        .toMatchSnapshot();
    });
});