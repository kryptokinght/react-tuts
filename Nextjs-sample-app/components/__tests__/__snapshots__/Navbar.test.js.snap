// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Navbar /> should render 1`] = `
<nav
  className="navbar navbar-expand navbar-dark bg-dark mb-4"
>
  <div
    className="container"
  >
    <a
      className="navbar-brand"
      href="#"
    >
      BitzPrice
    </a>
    <div
      className="collapse navbar-collapse"
    >
      <ul
        className="navbar-nav ml-auto"
      >
        <li
          className="nav-item"
        >
          <Link
            href="/"
          >
            <a
              className="nav-link"
            >
              Home
            </a>
          </Link>
        </li>
        <li
          className="nav-item"
        >
          <Link
            href="/about"
          >
            <a
              className="nav-link"
            >
              About
            </a>
          </Link>
        </li>
      </ul>
    </div>
  </div>
</nav>
`;
